sysdig/resource_sysdig_group_mapping_config_test.go

```diff
@@ -0,0 +1,78 @@
+//go:build tf_acc_sysdig || tf_acc_sysdig_monitor
+
+package sysdig_test
+
+import (
+	"github.com/draios/terraform-provider-sysdig/sysdig"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"os"
+	"testing"
+)
+
+func TestAccGroupMappingConfig(t *testing.T) {
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
```

Comment:
Change this PreCheck similar to this: https://github.com/sysdiglabs/terraform-provider-sysdig/blob/master/sysdig/resource_sysdig_monitor_alert_downtime_test.go#L20

We do not want to copy-paste this same block of code all over the place. This function can help us :)